spec.html

```diff
@@ -23106,7 +23106,7 @@ <h1>ECMAScript Standard Built-in Objects</h1>
   <p>There are certain built-in objects available whenever an ECMAScript |Script| or |Module| begins execution. One, the global object, is part of the lexical environment of the executing program. Others are accessible as initial properties of the global object or indirectly as properties of accessible built-in objects.</p>
   <p>Unless specified otherwise, a built-in object that is callable as a function is a built-in function object with the characteristics described in <emu-xref href="#sec-built-in-function-objects"></emu-xref>. Unless specified otherwise, the [[Extensible]] internal slot of a built-in object initially has the value *true*. Every built-in function object has a [[Realm]] internal slot whose value is the Realm Record of the realm for which the object was initially created.</p>
   <p>Many built-in objects are functions: they can be invoked with arguments. Some of them furthermore are constructors: they are functions intended for use with the `new` operator. For each built-in function, this specification describes the arguments required by that function and the properties of that function object. For each built-in constructor, this specification furthermore describes properties of the prototype object of that constructor and properties of specific object instances returned by a `new` expression that invokes that constructor.</p>
-  <p>Unless otherwise specified in the description of a particular function, if a built-in function or constructor is given fewer arguments than the function is specified to require, the function or constructor shall behave exactly as if it had been given sufficient additional arguments, each such argument being the *undefined* value. Such missing arguments are considered to be &ldquo;not present&rdquo; and may be identified in that manner by specification algorithms. In the description of a particular function, the terms &ldquo;`this` value&rdquo; and &ldquo;NewTarget&rdquo; have the meanings given in <emu-xref href="#sec-built-in-function-objects"></emu-xref>.</p>
+  <p>Unless otherwise specified in the description of a particular function, if a built-in function or constructor is given fewer arguments than the function is specified to require, the function or constructor shall behave exactly as if it had been given sufficient additional arguments, each such argument being the *undefined* value. Such missing arguments, if not covered by rest arguments, are considered to be &ldquo;not present&rdquo; and may be identified in that manner by specification algorithms. In the description of a particular function, the terms &ldquo;`this` value&rdquo; and &ldquo;NewTarget&rdquo; have the meanings given in <emu-xref href="#sec-built-in-function-objects"></emu-xref>.</p>
```

Comment:
I don't think it suffices to simply insert "if not covered by rest arguments", because we haven't defined 'rest argument' (or parameter) in the context of a built-in function's signature, or what it means to be "covered by" one.

Also,  I don't think you can delete specific steps like `Let _sources_ be the List of argument values starting with the second argument` without specifying the general mechanism instead.

So I think I'd like to see this paragraph expanded into a specification of the things that appear in built-ins' parameter lists (including square-brackets and ellipsis), and the way that parameters get their values for a given invocation (i.e., using values, if any, from the _argumentsList_ of the [[Call]]).  (This could possibly be a new clause under "Algorithm Conventions", or even something in [9.3.1](https://tc39.github.io/ecma262/#sec-built-in-function-objects-call-thisargument-argumentslist), but I think it might make more sense here.)